pages/community/foundation/index.tsx

```diff
@@ -224,8 +223,6 @@ export default function Governance() {
           }}
         />
       </Box>
-
-      <FoundationFAQ />
```

Comment:
Also, is this component safe to delete?

Reply:
Yup, all the content was moved to the Grants FAQ